addons/web/static/src/xml/notification.xml

```diff
@@ -14,7 +14,7 @@
                  t-raw="widget.title"/>
             <strong t-if="widget.subtitle" t-raw="widget.subtitle"
                 class="o_notification_subtitle mb-2 font-weight-normal text-muted"/>
-            <div t-if="widget.message" t-raw="widget.message"
+            <div t-if="widget.message || widget.contentEl" t-raw="widget.message"
```

Comment:
Dammit, you touched a t-raw line... :grimacing: https://runbot.odoo.com/runbot/build/6919587 :bomb:

I don't think the o_notification_content is needed for anything. Maybe no change in the XML and adding the contentEl in the toast body directly in JS?
Or setting "message" to a truthy value if contentEl is defined then replacing `this.$('.o_notification_content').append(this.contentEl);` by `this.$('.o_notification_content').empty().append(this.contentEl);`